src/PrestaShopBundle/Resources/config/routing/admin/improve/international/tax_rules_groups.yml

```diff
@@ -6,6 +6,11 @@ admin_tax_rules_groups_index:
     _legacy_controller: AdminTaxRulesGroup
 #    _legacy_link: AdminTaxRulesGroup
 
+# not sure why this route appears in the list of routes that are not following naming conventions
+# linter suggest to change it to: admin_commons_search_grid
```

Comment:
Some of our controllers use a common Search action. Other have a dedicated search action. I dont remember which practice is the latest, which one is the oldest. We need to explore that to know the answer whether or not the linter is right here.